src/app/lorax/directives/modal-issue.tpl.html

```diff
@@ -0,0 +1,28 @@
+<article class="modal modal--fade-show modal-issue" ng-show="modalIssue.open" >
+  <a href="#" class="modal__close modal__close-absolute icon-close" data-lorax-prevent-default ng-click="modalIssue.closeModal()">Close</a>
+  <div class="container">
+    <h1>{{modalIssue.issue.getName()}}</h1>
```

Comment:
AFAIK, screen-readers have yet to implement the new HTML5 document outline algorithm, so while multiple `<h1>`'s are valid per the official spec, they are still somewhat confusing for AT users.

Fine to leave this for now though, can always follow up when you have a look at the overall structure :+1: